apps/backend/src/modules/kobo/tests/kobo.controller.spec.ts

```diff
@@ -31,57 +26,56 @@ describe('KoboController', () => {
     const testingModule: TestingModule = await Test.createTestingModule({
       imports: [AppModule],
     }).compile();
-    userFactory = new UserFactory();
+    userRepository = testingModule.get(getRepositoryToken(User));
+    userFactory = new UserFactory(userRepository);
     authService = testingModule.get(AuthService);
     koboService = testingModule.get(KoboService);
     app = testingModule.createNestApplication();
     await app.init();
   });
 
+  afterEach(async () => {
+    await userRepository.clear();
+  });
```

Comment:
why do you have to do this

Reply:
To cleanup the repository (cf user controller tests)